IntegrationWithPharmacies/Services/ClientScheduledService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Grpc.Core;
+using IntegrationWithPharmacies.Protos;
+using System.Threading.Tasks;
+
+namespace IntegrationWithPharmacies
+{
+    public class ClientScheduledService
+    {
+        private Channel channel;
+        private SpringGrpcService.SpringGrpcServiceClient client;
+
+        public ClientScheduledService() { }
+
+        public async Task<string> SendMessage(string name)
+        {
+            channel = new Channel("127.0.0.1:8787", ChannelCredentials.Insecure);
+            client = new SpringGrpcService.SpringGrpcServiceClient(channel);
```

Comment:
Write this in form:
"client = new SpringGrpcService.SpringGrpcServiceClient(new Channel("127.0.0.1:8787", ChannelCredentials.Insecure));"